### ISSUE TITLE
Project does not converge

### ISSUE THREAD

**Issue by GitHubmalajava** [clear-nus/TactileSGNet#4]
@haroldsoh ，
Hi, Thanks for your contribution. I recently encountered a problem where the project fails to converge when I run main.py. Have you encountered a similar problem during training? 
Looking forward to your reply. Thank you
My output is as follows：
At epoch: 2, training_loss: 16.000000, test_loss: 4.000000, best accuracy: 2.778, time elasped: 380.012
100%|██████████| 576/576 [05:40<00:00,  1.69it/s]
100%|██████████| 576/576 [05:40<00:00,  1.69it/s]
At epoch: 4, training_loss: 16.000000, test_loss: 4.000000, best accuracy: 2.778, time elasped: 382.505
100%|██████████| 576/576 [05:39<00:00,  1.70it/s]
100%|██████████| 576/576 [05:37<00:00,  1.71it/s]
....
At epoch: 90, training_loss: 17.438541, test_loss: 4.302250, best accuracy: 2.778, time elasped: 416.339
100%|██████████| 576/576 [06:06<00:00,  1.57it/s]
100%|██████████| 576/576 [06:06<00:00,  1.57it/s]
At epoch: 92, training_loss: 17.468197, test_loss: 4.346666, best accuracy: 2.778, time elasped: 416.646
100%|██████████| 576/576 [06:07<00:00,  1.57it/s]
100%|██████████| 576/576 [06:08<00:00,  1.56it/s]
...

**Comment by fuqianggu** [comment]
Hi, Thank you for your interest. We didn't encounter such problem. You may need to check the version of related library and the values of hyperparameters.

**Comment by GitHubmalajava** [comment]
@fuqianggu ,
I'm confused about this, I get this error when I modify the learning rate or the optimizer, my anaconda environment ：pytorch = 1.4 ， python=3.6,cuda=10.1. I installed it according to your readme instructions. I'm confused, is there a problem with the torch_geometric of the graph convolution module.
Looking forward to your reply. Thank you.
